### PR TITLE
add post user and formated time

### PR DESCRIPTION
**Pull request by Angelica137**
We added the users' names to posts and formatted date.

### CODE REVIEW

File: src/components/Postlist.js [hunk 1]
```diff
@@ -40,8 +42,9 @@ class Postlist extends React.Component {
         <section>
           {posts.map((post) => (
             <article key={post.id}>
-              <p>{post.created_at}</p>
+              <p>{moment(post.created_at).format('MMMM Do YYYY, h:mm a')}</p>
```
Comment (FayeCarter) [inline src/components/Postlist.js:45]: nice

File: src/components/Postlist.js [hunk 2]
```diff
@@ -40,8 +42,9 @@ class Postlist extends React.Component {
...
               <p>{post.message}</p>
+              <p>{post.first_name}</p>
```
Comment (FayeCarter) [inline src/components/Postlist.js:47]: Success! 👍🏻 🤝

**Comment by FayeCarter** [comment]
I like this code

File: src/components/Postlist.js (hunk 2 above)
Comment (Steven-Klavins) [reply src/components/Postlist.js:46]: Awesome stuff

**Pull request merged**